src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -558,6 +665,37 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                         }
                     }
                 });
+
+                // squeezeComponent is a new member on the controllerData
+                controllerData.squeezeComponent = motionController.getComponent("grasp");
+
+                // onSqueezeButtonChangedObserver is also a new variable
+                if(controllerData.squeezeComponent)
+                {
+                    controllerData.onSqueezeButtonChangedObserver = controllerData.squeezeComponent.onButtonStateChangedObservable.add((component) => {
+                        if (component.changes.pressed) {
+                            const pressed = component.changes.pressed.current;
+                            if (controllerData.pick) {

```

Comment:
Won't the "grasp" take care of filtering differences between nearTouch vs nearGrab? I assumed this will be triggered for grab and not for touch. If that is not the case, I should definitely have a filter